runtime/laos/src/configs/cumulus_parachain_system.rs

```diff
@@ -39,11 +39,27 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type XcmpMessageHandler = XcmpQueue;
 	type ReservedXcmpWeight = ReservedXcmpWeight;
 	type CheckAssociatedRelayNumber = cumulus_pallet_parachain_system::RelayNumberStrictlyIncreases;
-	type ConsensusHook = cumulus_pallet_parachain_system::ExpectParentIncluded;
+	type ConsensusHook = ConsensusHook;
```

Comment:
Please, add the note regarding `ExpectParentIncluded`, as this type "handles the migration to asynchronous backing gracefully", as per polkadot-sdk docs -> https://docs.rs/cumulus-pallet-parachain-system/latest/cumulus_pallet_parachain_system/consensus_hook/struct.ExpectParentIncluded.html and merge the PR